SampleApps/README.md

```diff
@@ -0,0 +1,8 @@
+# Sample apps and Deployment samples
+
+<!-- only enough info to differentiate this dir level vs the others; what is different about this dir compared to the sibling dirs? -->
+This directory contains:
+
+*  Sample apps - WebView2 sample apps for various frameworks and platforms, as Visual Studio projects.  For more information, see [Sample apps](https://docs.microsoft.com/microsoft-edge/webview2/code-samples-links).
+
+*  Deployment samples - Samples that demonstrate deploying the WebView2 Runtime.  For more information, see [Deployment samples](https://docs.microsoft.com/microsoft-edge/webview2/samples/deployment-samples).
```

Comment:
See also: 

There's another WebView2 sample repo: [WebView2Browser repo](https://github.com/MicrosoftEdge/WebView2Browser) - uses multiple WebView2 instances in a single Win32 application.

WebView2Browser is a sample Windows desktop application demonstrating the capabilities of the WebView2 control.  <!-- delta: added sentence: -->The WebView2Browser sample app uses multiple WebView2 instances. 
A Win32 Visual Studio 2019 project using C++ and JavaScript.

Reply:
@nishitha-burman - We can possibly do nothing and Resolve Conversation.  A work item suggests moving the Browser sample into this repo, and the regular docs that the redesigned readmes link to will have a link to the Browser sample.  

Trying to avoid maintaining lists of individual samples in high-level readmes.

Reply:
The Samples nav page in the regular docs has a table row for WebView2Browser.
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/code-samples-links?branch=pr-en-us-2069

Reply:
@nishitha-burman 
Confirmed: The Samples nav page in the regular docs has a table row for WebView2Browser.
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/code-samples-links?branch=pr-en-us-2069

Propose to keep PR 140 top-level readme not mentioning WebView2Browser, & keep Docs PR 2069, then move WebView2Browser into this WebView2Samples repo.